tests/test_http.sh

```diff
@@ -24,8 +24,36 @@ rlJournalStart
         assert_up_and_running
     rlPhaseEnd
 
+    rlPhaseStartTest "Should not display SSL warning for HTTPS connection"
+        rlRun -t -c "docker exec -i kiwi_web /Kiwi/manage.py createsuperuser \
+            --username testadmin --email testadmin@domain.com --noinput"
+        rlRun -t -c "cat tests/set_testadmin_pass.py | docker exec -i kiwi_web /Kiwi/manage.py shell"
+        IP_ADDRESS=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' kiwi_web`
+        rm -f /tmp/testcookies.txt
+        rm -f /tmp/testdata.sslon
```

Comment:
Do you really need to remove those files or they get overwritten ?

Reply:
It's not clearly stated in the curl documentation. Currently the output file is always overwritten, the cookie file is created / overwritten when -c option is used.